README.md

```diff
@@ -154,43 +154,38 @@ Each task can hold 0...n results by each *worker* defined in the task's `to` fie
 A succeeded result for the above task:
 ```json
 {
-  "id": "8db76400-e2d9-4d9d-881f-f073336338c1",
   "from": "app1.proxy-hd.broker-project1.samply.de",
   "to": [
     "app7.proxy-hd.broker-project1.samply.de"
   ],
   "task": "70c0aa90-bfcf-4312-a6af-42cbd57dc0b8",
-  "status": {
-    "succeeded": "Successfully quenched 1.43e14 flux pulse devices"
-  },
+  "status": "succeeded",
+  "body": "Successfully quenched 1.43e14 flux pulse devices",
   "metadata": ["Arbitrary", "types", "are", "possible"]
 }
 ```
 
 A failed task:
 ```json
 {
-  "id": "24a49494-6a00-415f-80fc-b2ae34658b98",
   "from": "app5.proxy-ma.broker-project1.samply.de",
   "to": [
     "app7.proxy-hd.broker-project1.samply.de"
   ],
   "task": "70c0aa90-bfcf-4312-a6af-42cbd57dc0b8",
-  "status": {
-    "permfailed": "Unable to decrypt quantum state"
-  },
+  "status": "permfailed",
+  "body": "Unable to decrypt quantum state",
   "metadata": {
     "complex": "A map (key 'complex') is possible, too"
   }
 }
 ```
 
-- `id`: UUID identifying the result. Note that when the result is initially submitted, the server is not required to use the submitted ID but may auto-generate its own one. Currently, since there can be only 0..1 results per client (= `from` field), a result's URL has the form `/v1/tasks/<task_id>/results/<id_in_from_field>` and the `id` field is only used internally, e.g. for filtering. However, for future compatibility, Callers must assume the submission's `id` property is ignored and check the reply's `Location` header for the actual URL to the task.
 - `from`: BeamID identifying the client submitting this result. This needs to match an entry the `to` field in the task.
 - `to`: BeamIDs the intended recipients of the result. Used for encrypted payloads.
 - `task`: UUID identifying the task this result belongs to.
-- `status`: Defines status of this work result. Possible values `claimed`, `tempfailed(<body>)`, `permfailed(<body>)`, `succeeded(<body>)`. It is up to the application how these statuses are used. For example, some application might require workers to acknowledge the receipt of tasks by setting `status=claimed`, whereas others have only short-running tasks and skip this step.
-- `status.body`: Required for `status`es listed above with `(<body>)`. Either carries the actual result payload of the task (`succeeded`) or an error message.
+- `status`: Defines status of this work result. Allowed values `claimed`, `tempfailed`, `permfailed`, `succeeded`. It is up to the application how these statuses are used. For example, some application might require workers to acknowledge the receipt of tasks by setting `status=claimed`, whereas others have only short-running tasks and skip this step.
+- `body`: Required for all `status`es except for `claimed`. Either carries the actual result payload of the task in case the status is `succeeded` or an error message.
```

Comment:
```suggestion
- `body`: Supported and required for all `status`es except for `claimed`. Either carries the actual result payload of the task in case the status is `succeeded` or an error message.
```